htdocs/editicons.bml

```diff
@@ -117,32 +119,7 @@ use strict;
 
             my $MAX_UPLOAD = LJ::Userpic->max_allowed_bytes($u);
             my $error;
-
-            # act like the factory is disabled if no mogile
-            my $factory_disabled = !LJ::is_enabled('userpicfactory') || !LJ::mogclient();
```

Comment:
IIRC (and this is just vague memory) the userpic factory requires some other modules to be installed and potentially can take up a lot of disk space, and it was also a little bit buggy at first when it was added, so it was set up as an is_enabled so that LJ clones that didn't want to use it could turn it off.